app/components/publications/publication/case/remark/remark-panel.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+
+export default class PublicationsPublicationCaseRemarkPanelComponent extends Component {
+  @tracked isInEditMode;
+
+  @tracked remark;
+
+  constructor() {
+    super(...arguments);
+  }
+
+  @action
+  putInEditMode() {
+    const publicationFlow = this.args.publicationFlow;
+    this.isInEditMode = true;
+    this.remark = publicationFlow.remark;
+  }
+
+  @action
+  cancelEdit() {
+    this.showError = false;
+    this.isInEditMode = false;
+  }
+
+  @task
+  *save() {
+    const publicationFlow = this.args.publicationFlow;
+    publicationFlow.remark = this.remark;
+    // no try-catch: don't exit if save didn't work
```

Comment:
Als de save niet gewerkt heeft, moet `publicationFlow.remark` wel gerollbacked worden. Anders wordt dat misschien mee opgeslagen bij het opslaan van een andere panel.

Reply:
Eigenlijk hebt ge daar gelijk, maar is dat niet te veel uitzonderingen opvangen?

Reply:
Ja, waarschijnlijk wel. Want dan zouden we dat ook op het andere panel moeten doen en daar is de rollback-strategie al veel complexer omdat er meerdere saves/promises zijn. Negeer deze comment dus maar.